app/components/post_list/post/post.tsx

```diff
@@ -137,7 +138,7 @@ const Post = ({
             const isValidSystemMessage = isAutoResponder || !isSystemPost;
             if (post.deleteAt !== 0 && isValidSystemMessage && !isPendingOrFailed) {
                 if ([Screens.CHANNEL, Screens.PERMALINK].includes(location)) {
-                    DeviceEventEmitter.emit('goToThread', post);
+                    DeviceEventEmitter.emit(Events.GO_TO_THREAD, post);
```

Comment:
Why the navigation is done by event instead of by go to screen or something similar?

Reply:
The reality is that I don't remember why we needed to handle it that way on v1. That said qe have't included the thread screen yet, so at least this is avoiding a crash for now hehhehe